src/crawlee/basic_crawler/basic_crawler.py

```diff
@@ -202,6 +202,10 @@ def __init__(
             root_logger.addHandler(handler)
             root_logger.setLevel(logging.INFO if not sys.flags.dev_mode else logging.DEBUG)
 
+            # Silence HTTPX logger
+            httpx_logger = logging.getLogger('httpx')
+            httpx_logger.setLevel(logging.WARNING)
```

Comment:
Can we make it more verbose with a toggle? Perhaps based on `sys.flags.dev_mode`? Also, I'm not thrilled about `BasicCrawler.__init__` changing global state (log levels), but from a quick glance, we cannot help that.